mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java

```diff
@@ -363,7 +371,10 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                     startCrop(selectedImageUri);
                 }
             } else if (requestCode == UCrop.REQUEST_CROP) {
-                handleCropResult(data);
+                try {
+                    handleCropResult(data);
+                } catch (IOException e) {
```

Comment:
Exception not handled correctly.

Reply:
So should i simply log the Error and proceed